source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -46,12 +46,19 @@ public void go() {
     private void nearExample() {
         // begin findExample
 
-        // code to set up your mongo client ...
+        // Add your MongoCollection setup code here
         MongoDatabase database = mongoClient.getDatabase("sample_mflix");
         MongoCollection<Document> collection = database.getCollection("theaters");
+
         Point centralPark = new Point(new Position(-73.9667, 40.78));
+        
+        // Creates a query that matches all theaters between 5,000 and 10,000 meters from the specified Point
         Bson query = near("location.geo", centralPark, 10000.0, 5000.0);
+
+        // Creates a projection to include only the "location.address.city" field in the results
         Bson projection = fields(include("location.address.city"), excludeId());
+        
+        // Prints the projected field of the results from the geolocation query as JSON
```

Comment:
Issue:
I think the correct term is [geospatial query](https://www.mongodb.com/docs/manual/geospatial-queries/#geospatial-queries-1) as described by the Server docs.


Suggestion:
I think it would be good to use the correct terminology in all instances.